_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+
+### Audio outputs
+The Hardware Ouputs section in the top right should have A1, A2, A3 with a triangle beside them. Each one of these have a selectable menu for our output devices. Each input device has a toggle-able setting for us to push audio toe each individual output device, which will get  better explanation when we get to inputs inputs. For now, click on A1 and select your headphones. click on A2 and select the virtual audio cable.
+
+Your A1, A2, and A3 section should look a little something like this:
+
+![https://i.imgur.com/Y5BvneM.png](https://i.imgur.com/Y5BvneM.png)
+
+Below that you will see a bunch of vertical bars, for now we will leave these alone until we have audio pushing through them. For that, we will start by pushing computer audio to them through a digital inputs.
+
+
+### Digital Inputs and Windows Audio
+For digital inputs we will be looking at the middle columns of Voicemeeter Banana which should look a little something like this:
+
+Voicemeeter Digital Inputs:
+
+![https://i.imgur.com/pkMT14v.png](https://i.imgur.com/pkMT14v.png)
+
+To set our Windows Default Audio to push through these digital inputs we need to open Windows Sound Settings by right clicking on the speaker icon that adjusts our audio in the task bar. After right clicking select 'Open Sound Settings' and you should see a window with
+
+Sound-
+Choose your output device:
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+Select VoiceMeeter Input (VB-Audio Voicemeeter VAIO) as the default
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+After setting VB-Audio Voicemeeter VAIO as our default output device pull up Voicemeeter Banana. In the middle under the Voicemeeter VAIO section we should see the audio levels rising and lowering if you play sounds on our computer. Do you? Great! If not? Try again!
+
+You may or may not be able to hear these right away, if not just take a look in the column down at the A1, A2, A3 and select the ones for the headphones. If you set up along with the tutorial it should be A1. At this point you should be able to hear the audio going through your headphones which will show in the bottom right level bars under the A1 column.
+
+Bottom right Hardware Outputs audio with audio pushed through.
+
+You will notice this has a bunch of different options, including a mono toggle and a mute toggle! At this point you should be hearing your computer audio going through and being pushed to your headphones.
+
+
+### Hardware inputs and microphone
+Are you ready to set up a microphone? I am! Alright, so microphones. The far left section, Hardware Inputs, there should be a 'Select your device' listing, click this and you should get an option between a bunch of different things. Select the WDM version of your microphone for now, we can get into different versions of a device driver later but for now just use WDM.
```

Comment:
Add "In" to the start of "The far left section" and lowercase the "The". Replace the comma after "listing" with a period and capitalize "Click".

Instead of saying "we can get into different versions of a device driver", either say "we will discuss the different versions of a device driver later in this article, but for now just use WDM" OR "we will discuss the different versions of a device driver in a different article, but for now just use WDM". This makes it a bit more explicit on when to expect that information.

Reply:
revised to say we will discuss different versions of the drivers in a different article. I'd like to research that a bit more and it isn't mandatory for setup right away. More of a deeper dive thing I believe.